modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -135,36 +135,39 @@ kind: MachineConfig
 metadata:
   labels:
     machineconfiguration.openshift.io/role: worker
-  name: 98-var-log-partition
+  name: 98-var-partition
 spec:
   config:
     ignition:
       version: 3.1.0
     storage:
       disks:
-      - device: /dev/nvme0n1
+      - device: /dev/<device_name> <1>
         wipeTable: false
         partitions:
-        - sizeMiB: 47000
-          startMiB: 47000
-          label: var-log
+        - sizeMiB: <partition_size> <2>
+          startMiB: <partition_start_size> <2>
+          label: var
       filesystems:
-        - path: /var/log
-          device: /dev/disk/by-partlabel/var-log
+        - path: /var
+          device: /dev/disk/by-partlabel/var
           format: xfs
     systemd:
       units:
-        - name: var-log.mount
+        - name: var.mount
           enabled: true
           contents: |
             [Unit]
             Before=local-fs.target
             [Mount]
-            Where=/var/log
-            What=/dev/disk/by-partlabel/var-log
+            Where=/var
+            What=/dev/disk/by-partlabel/var
             [Install]
             WantedBy=local-fs.target
 ----
++
+<1> The storage device name of the disk that you want to partition.
+<2> You must specify at least a `startMiB` value in mebibytes, such as `47000`, when partitioning onto the same disk as `sysroot`. If you do not, the partition might overtake your `sysroot` folder.
```

Comment:
This doesn't seem very clear to me.  What actually happens is that the root filesystem gets resized to fill the space up to the beginning of the data partition.  So, the `startMiB` effectively sets the size of the root filesystem, after a couple GiB of of overhead are subtracted off.  If no value is specified, or the specified value is too small, future reinstalls might overwrite the beginning of the data partition.

We should specify a minimum recommended `startMiB` value.  Perhaps 25000 or 50000?

Reply:
I removed the callout for `sizeMiB` and left it for only `startMiB` with the following callout note based on @bgilbert's comments:

> <2> A minimum of 25000 mebibytes is recommended. To fill data partition space, `startMiB` sets the size of the root filesystem. If no value is specified, or if the specified value is too small, future reinstalls might overwrite the beginning of the data partition.

OK?

@dustymabe @jlebon @cgwalters @yuqi-zhang

Reply:
I removed the callout for `sizeMiB` and left it for only `startMiB` with the following callout note based on @bgilbert's comments:

> <2> A minimum of 25000 mebibytes is recommended. To fill data partition space, `startMiB` sets the size of the root filesystem. If no value is specified, or if the specified value is too small, future reinstalls might overwrite the beginning of the data partition.

OK?

@dustymabe @jlebon @cgwalters @yuqi-zhang